lightning/src/ln/channelmanager.rs

```diff
@@ -2948,7 +2948,7 @@ where
 
 			outbound_scid_aliases: Mutex::new(new_hash_set()),
 			pending_inbound_payments: Mutex::new(new_hash_map()),
-			pending_outbound_payments: OutboundPayments::new(),
+			pending_outbound_payments: OutboundPayments::new(HashMap::new()),
```

Comment:
You'll need to use `new_hash_map()` here and everywhere else in our codebase.

Reply:
Alright! Thanks a lot for the pointer!

Fixed in [pr3010.17](https://github.com/shaavan/rust-lightning/commits/pr3010.17)